core/src/main/scala/org/broadinstitute/dsde/rawls/util/TracingUtils.scala

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.dsde.rawls.util
+
+import io.opencensus.scala.Tracing
+import io.opencensus.scala.Tracing._
+import io.opencensus.trace.Status
+import org.broadinstitute.dsde.rawls.model.{RawlsRequestContext, RawlsTracingContext}
+import slick.dbio.{DBIO, DBIOAction, Effect, NoStream}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object TracingUtils {
```

Comment:
This was `OpenCensusDBIOUtils` but was reformulated to be in terms of `RawlsRequestContext` and `RawlsTracingContext`